### PR TITLE
Add Search by Speaker ID and Member ID to Speakers

### PR DESCRIPTION
**Pull request by romanetar**
ref https://tipit.avaza.com/project/view#!tab=task-pane&groupby=MyTaskDueDate&view=vertical&task=3008859&fileview=grid

Signed-off-by: romanetar <roman_ag@hotmail.com>

### CODE REVIEW

File: app/Repositories/Summit/DoctrineSpeakerRepository.php [hunk 1]
```diff
@@ -905,6 +913,7 @@ public function getAllByPage(PagingInfo $paging_info, Filter $filter = null, Ord
 SELECT COUNT(DISTINCT(ID)) AS QTY
 FROM (
 	SELECT S.ID,
+    M.ExternalUserID AS MemberUserExternalID,
```
Comment (smarcet) [inline app/Repositories/Summit/DoctrineSpeakerRepository.php:916]: please align spaces it and add MemberID to count query

**Review by smarcet — CHANGES_REQUESTED** [review]
@romanetar please review comments

File: app/Repositories/Summit/DoctrineSpeakerRepository.php (hunk 1 above)
Comment (romanetar) [reply app/Repositories/Summit/DoctrineSpeakerRepository.php]: done

**Comment by romanetar** [comment]
@smarcet review resolved. Much thanks

**Review by smarcet — APPROVED** [review]
LGTM

**Pull request merged**